web/client/components/data/featuregrid/toolbars/TButton.jsx

```diff
@@ -15,15 +17,26 @@ const hideStyle = {
 };
 const normalStyle = {};
 const getStyle = (visible) => visible ? normalStyle : hideStyle;
-
-export const SimpleTButton = ({disabled, id, visible, onClick, glyph, active, className = "square-button", ...props}) => {
-    return (<Button {...props} bsStyle={active ? "success" : "primary"} disabled={disabled} id={`fg-${id}`}
```

Comment:
@offtherailz It's only a guess but couldn't be the issue due to ref?
here there is a spread of props that in case of class it could include and work also with refs while with function we need to explicit the ref with forwardRef

```js
import React, { forwardRef } from 'react';
...
export const SimpleTButton = forwardRef(({disabled, id, visible, onClick, glyph, active, className = "square-button", ...props}), ref) => {
    return (<Button {...props} ref={ref} bsStyle={active ? "success" : "primary"} disabled={disabled} id={`fg-${id}`}
```

but maybe it's not direclty related to this

Reply:
Thank you :+1: , I did it and it seems to work, waiting for a review and travis build to merge